src/moped/src/main/java/core/sensors/LowPassFilter.java

```diff
@@ -0,0 +1,25 @@
+package core.sensors;
```

Comment:
We should move this into another package, maybe data_generators? and not package this with the product. :)